src/jsonSass.js

```diff
@@ -22,5 +22,6 @@ function jsonSass(options) {
 }
 
 module.exports = jsonSass;
+module.exports.convertJs = jsToSassString;
 export default jsonSass;
 export { jsToSassString as convertJs };
```

Comment:
this line should be exporting it, I believe. I wonder if line 24 is causing isses with it. Babel handles translating ES6 module syntax to requirejs.